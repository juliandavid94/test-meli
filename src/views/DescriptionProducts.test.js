import { render } from "@testing-library/react";
import DescriptionProduct from "./DescriptionProduct";
import fetchMock from 'jest-fetch-mock';

describe("DescriptionProduct", () => {
    
    it('renders', async() => {
        fetchMock.mockResponseOnce(JSON.stringify({ author:{name: 'julian'} }));
        const categories = ['computacion'];
        const {container} = render(<DescriptionProduct data={categories}/>);
        expect(container).toMatchSnapshot();
    });
});