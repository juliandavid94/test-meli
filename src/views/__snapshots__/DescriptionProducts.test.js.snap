// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DescriptionProduct renders 1`] = `<div />`;
